src/Uno.UWP/Globalization/NumberFormatting/NumeralSystemTranslatorHelper.cs

```diff
@@ -0,0 +1,3566 @@
+					return "vi";
+				case "vi-vn":
+					return "vi";
+				case "vo":
+					return "vo";
+				case "vo-001":
+					return "vo-Latn-001";
+				case "vun":
+					return "vun";
+				case "vun-tz":
+					return "vun-Latn-TZ";
+				case "wae":
+					return "wae";
+				case "wae-ch":
+					return "wae-Latn-CH";
+				case "cy":
+					return "cy";
+				case "cy-gb":
+					return "cy";
+				case "wal":
+					return "wal";
+				case "wal-et":
+					return "wal-Ethi-ET";
+				case "wo":
+					return "wo-Latn";
+				case "wo-sn":
+					return "wo-Latn";
+				case "xh":
+					return "xh";
+				case "xh-za":
+					return "xh";
+				case "yav":
+					return "yav";
+				case "yav-cm":
+					return "yav-Latn-CM";
+				case "ii":
+					return "ii";
+				case "ii-cn":
+					return "ii-Yiii";
+				case "yi-001":
+					return "yi";
+				case "yo":
+					return "yo";
+				case "yo-bj":
+					return "yo-BJ";
+				case "yo-ng":
+					return "yo-Latn";
+				case "dje":
+					return "dje";
+				case "dje-ne":
+					return "dje-Latn-NE";
+				case "zu":
+					return "zu";
+				case "zu-za":
+					return "zu";
+				default:
+					throw new ArgumentException("The parameter is incorrect.");
+			}
+		}
+
+		public static char[] GetDigitsSource(string numeralSystem)
+		{
+			switch (numeralSystem)
+			{
+				case "Arab":
+					return ArabDigits;
+				case "ArabExt":
+					return ArabExtDigits;
+				case "Bali":
+					return BaliDigits;
+				case "Beng":
+					return BengDigits;
+				case "Cham":
+					return ChamDigits;
+				case "Deva":
+					return DevaDigits;
+				case "FullWide":
+					return FullWideDigits;
+				case "Gujr":
+					return GujrDigits;
+				case "Guru":
+					return GuruDigits;
+				case "Java":
+					return JavaDigits;
+				case "Kali":
+					return KaliDigits;
+				case "Khmr":
+					return KhmrDigits;
+				case "Knda":
+					return KndaDigits;
+				case "Lana":
+					return LanaDigits;
+				case "LanaTham":
+					return LanaThamDigits;
+				case "Laoo":
+					return LaooDigits;
+				case "Latn":
+					return LatnDigits;
+				case "Lepc":
+					return LepcDigits;
+				case "Limb":
+					return LimbDigits;
+				case "Mlym":
+					return MlymDigits;
+				case "Mong":
+					return MongDigits;
+				case "Mtei":
+					return MteiDigits;
+				case "Mymr":
+					return MymrDigits;
+				case "MymrShan":
+					return MymrShanDigits;
+				case "Nkoo":
+					return NkooDigits;
+				case "Olck":
+					return OlckDigits;
+				case "Orya":
+					return OryaDigits;
+				case "Saur":
+					return SaurDigits;
+				case "Sund":
+					return SundDigits;
+				case "Talu":
+					return TaluDigits;
+				case "TamlDec":
+					return TamlDecDigits;
+				case "Telu":
+					return TeluDigits;
+				case "Thai":
+					return ThaiDigits;
+				case "Tibt":
+					return TibtDigits;
+				case "Vaii":
+					return VaiiDigits;
+				default:
+					throw new ArgumentException("The parameter is incorrect.");
+			}
+		}
+
+		public static string ToPascalCase(string numeralSystem)
```

Comment:
@jeromelaban I Added this method because NumeralSystem property is case insensitive but It is supposed to display NumeralSystem in a standard way.